src/applications/representative-appoint/containers/PreSubmitInfo.jsx

```diff
@@ -0,0 +1,154 @@
+  const representativeName = getRepresentativeName(formData);
+
+  useEffect(
+    () => {
+      if (termsAndConditionsChecked && formReplacementChecked) {
+        onSectionComplete(true);
+      }
+      return () => {
+        onSectionComplete(false);
+      };
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [termsAndConditionsChecked, formReplacementChecked],
+  );
+
+  useEffect(
+    () => {
+      if (showError && !hasSubmit) {
+        setTermsAndConditionsError(!termsAndConditionsChecked);
+        setFormReplacementError(!formReplacementChecked);
+      }
+    },
+    [showError, hasSubmit, termsAndConditionsChecked, formReplacementChecked],
+  );
+
+  if (isSubmitPending) {
+    return (
+      <div className="vads-u-margin-bottom--3">
+        <va-loading-indicator
+          label="Loading"
+          message="We’re processing your form"
+        />
+      </div>
+    );
+  }
+
+  return (
+    <>
+      {' '}
+      <h3>Agree to the terms and conditions</h3>
+      <va-accordion open-single>
+        <va-accordion-item
+          header="Our terms and conditions"
+          id="terms-and-conditions"
+        >
+          <p>
+            I, the claimant {applicantFullName} hereby appoint{' '}
+            {representativeName} as my representative to prepare, present and
+            prosecute my claim for any and all benefits from the Department of
+            Veterans Affairs (VA) based on the service of the veteran named.{' '}
+          </p>{' '}
+          <p>
+            I authorize the Department of Veterans Affairs to release any and
+            all of my records, to include disclosure of my Federal tax
+            information (other than the items identified in the permission
+            sections), to that service organization appointed as my
+            representative. It is understood that no fee or compensation of
+            whatsoever nature will be charged me for service rendered pursuant
+            to this power of attorney.
+          </p>
+          <p>
+            I understand that the service organization I have appointed as my
+            representative may revoke this power of attorney at any time,
+            subject to 38 CFR 20.608. Additionally, in those cases where a
+            Veteran’s income is being developed because of an income
+            verification necessitated by an Internal Revenue Service
+            verification match, the assignment of the service organization as
+            the Veteran’s representative is only valid for five years from the
+            date this form is signed for purposes restricted to the verification
+            match. Signed and accepted subject to the foregoing conditions.
+          </p>
+        </va-accordion-item>
+      </va-accordion>
+      <div className="vads-u-margin-top--neg4">
+        <VaCheckboxGroup>
+          {' '}
+          <VaCheckbox
```

Comment:
These checkboxes throw console warning for me locally:

> ncaught TypeError: Cannot read properties of undefined (reading 'length')
    at push../node_modules/@department-of-veterans-affairs/web-components/dist/esm-es5/va-checkbox.entry.js.__spreadArray (va-checkbox.entry.js:1:61)
    at e.fireAnalyticsEvent (va-checkbox.entry.js:1:16136)
    at HTMLInputElement.handleChange (va-checkbox.entry.js:1:16677)